src/appkit.cs

```diff
@@ -12969,7 +13237,7 @@ interface NSSearchFieldCell {
 	}
 	
 	[BaseType (typeof (NSControl))]
-	interface NSSegmentedControl {
+	interface NSSegmentedControl : INSUserInterfaceCompression {
```

Comment:
You want to use `NSUserInterfaceCompression` instead or generator will pick the wrong interface.